commitlint.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
please remove unnecessary file after we removed commit linting from pre commit hooks